crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -14,9 +14,13 @@
 
 #![allow(unused_imports)]
 
-use digest::{FixedOutput, InvalidLength, KeyInit, Output, Update};
+use std::ptr::hash;
+
+use digest::{FixedOutput, InvalidLength, KeyInit, Mac, Output, Update};
 use generic_array::GenericArray;
 use wasefire_applet_api::crypto::hash::{self as api, Algorithm, Api};
+use wasefire_applet_api::crypto::Api::Hash;
+use wasefire_board_api::crypto::{HashApi, HmacApi, LastError};
```

Comment:
You can remove this line (see below why).

Actually, you shouldn't need to touch any import.